src/version.h

```diff
@@ -10,9 +10,9 @@
 
 #pragma once
 
-#define REDISEARCH_VERSION_MAJOR 99
-#define REDISEARCH_VERSION_MINOR 99
-#define REDISEARCH_VERSION_PATCH 99
+#define REDISEARCH_VERSION_MAJOR 2
+#define REDISEARCH_VERSION_MINOR 8
+#define REDISEARCH_VERSION_PATCH 0
```

Comment:
Should not be part of this PR
(master should remain with version 99.99.99)

Reply:
I didn't touch this section either, strange